packages/atlas/src/views/global/YppLandingView/YppLandingView.hooks.ts

```diff
@@ -34,27 +34,24 @@ export type YppSyncedChannel = {
 
 export const useGetYppSyncedChannels = () => {
   const { activeMembership, membershipsLoading, isAuthLoading, channelId } = useUser()
-  const [isLoading, setIsLoading] = useState(true)
   const location = useLocation()
-  const [syncedChannels, setSyncedChannels] = useState<YppSyncedChannel[]>([])
+  const { data: syncedChannels, isLoading, refetch } = useQuery('ypp-synced-channels', () => getSyncedChannels())
```

Comment:
note that by default react-query will refetch all the queries every time user re-enters the tab. This could cause higher server load if the number of active queries is high. You can read about it [here](https://react-query-v3.tanstack.com/guides/window-focus-refetching)